### ISSUE TITLE
Support controller as or bindToController

### ISSUE THREAD

**Issue by tinusn** [troch/angular-multi-step-form#1]
It would be nice to support the controllerAs syntax (or set bindToController on the directive) so that you do not have to use $scope in controllers.

**Comment by troch** [comment]
Thanks for raising this issue.
- `controllerAs` is fairly straight forward to support.
- `bindToController`, I have to do a little bit more research as I am not familiar with its use. Although that would mean no possible support for Angular 1.2 (I have not tested against it yet, but I am thinking of trying to support it).

Currently, the directive and steps scopes are augmented with control functions: $nextStep(), $prevStep(), etc... Would you see them as being under the controllerAs variable or directly in scope?

To illustrate, let's say I have a step configured that way:

``` javascript
{
    controller: function () {
        this.name = '';
    },
    controllerAs: 'step'
}
```

**Choice 1:** control functions not under controllerAs reference (this is my preferred choice as the controllerAs syntax is designed to keep data separated, it would be counter-productive in my mind to add things against it which doesn't belong to it).

``` html
<!-- Inside step -->
<input type="text" ng-model="step.name">

<button ng-click="$nextStep()">Next</button>
```

**Choice 2:**

``` html
<!-- Inside step -->
<input type="text" ng-model="step.name">

<button ng-click="step.$nextStep()">Next</button>
```

**Comment by tinusn** [comment]
I would go for 1 - much simpler to understand and use.

**Comment by troch** [comment]
I have published v1.0.5. I'll do more research for `bindToController`, so far what I have found is applicable to directives with isolated scopes (and I don't have an isolated scope on the main directive). But I'll keep digging.

**Comment by tinusn** [comment]
Forget about the bindToController - I was looking to much at your directive...

The fix you provided works beatifully. Thank you for that!
